api/buildcraft/api/transport/IStripesHandler.java

```diff
@@ -17,6 +17,7 @@
 public interface IStripesHandler {
 	public static enum StripesHandlerType {
 		ITEM_USE,
+		ITEM_USE_LAST,
```

Comment:
Yeah, I agree. The other option I considered is splitting handlers into two lists, high priority handlers and low priority handlers.